packages/create-app/src/lib/tasks.ts

```diff
@@ -236,3 +244,79 @@ export async function moveAppTask(
       });
   });
 }
+
+/**
+ * Read git configs by creating a temp folder and initializing a repo
+ *
+ * @throws if `exec` fails
+ */
+export async function readGitConfig(): Promise<GitConfig | undefined> {
+  const tempDir = resolvePath(os.tmpdir(), 'git-temp-dir');
+
+  const runCmd = (cmd: string) =>
+    exec(cmd).catch(error => {
+      process.stdout.write(error.stderr);
+      process.stdout.write(error.stdout);
+      throw new Error(`Could not execute command ${chalk.cyan(cmd)}`);
+    });
+
+  const isGitAvailable = await commandExists('git').catch(() => false);
+
+  if (!isGitAvailable) return;
+
+  try {
+    await fs.mkdir(tempDir);
+
+    process.chdir(tempDir);
+
+    const [gitUsername, gitEmail] = await Promise.all([
+      runCmd('git config user.name'),
+      runCmd('git config user.email'),
+    ]);
```

Comment:
We don't actually need these right? Could simply skip this and that should hopefully fix the Linus E2E tests

Reply:
It's going to throw a error if we run git init without a git identity set.

Reply:
It throws an error if we commit without a git identity. That's why I'm making sure that there is a username and an email set.

Reply:
Alright, makes sense to grab these then